### PR TITLE
Add help.osohq.com messages.

### PR DESCRIPTION
**Pull request by dhatch**
These changes add links to a help endpoints that we can use to direct users to the appropriate resource for the type of error they are encountering.

### CODE REVIEW

File: languages/python/oso/oso/oso.py [hunk 1]
```diff
@@ -77,3 +79,10 @@ def get_allowed_actions(self, actor, resource, allow_wildcard=False) -> list:
             actions.add(action)
 
         return list(actions)
+
+    def _print_polar_log_message(self):
+        if os.environ.get("POLAR_LOG", None):
+            print(
+                "It looks like you've enabled Polar tracing. Get help with "
```
Comment (saolsen) [inline languages/python/oso/oso/oso.py:86]:
```suggestion
                "Polar tracing enabled. Get help with "
```

**Review by saolsen — APPROVED** [review]

File: languages/python/oso/oso/oso.py (hunk 1 above)
Comment (dhatch) [reply languages/python/oso/oso/oso.py:86]: good call

File: languages/python/oso/oso/oso.py [hunk 2]
```diff
@@ -77,3 +79,10 @@ def get_allowed_actions(self, actor, resource, allow_wildcard=False) -> list:
             actions.add(action)
 
         return list(actions)
+
+    def _print_polar_log_message(self):
+        if os.environ.get("POLAR_LOG", None):
+            print(
+                "Polar tracing enabled. Get help with "
```
Comment (gj) [inline languages/python/oso/oso/oso.py:86]: Doesn't have to get figured out as part of this PR, but should this maybe be `OSO_LOG=1` and `Oso tracing enabled.` in light of our attempts to standardize on Oso > Polar in public?

Comment (gneray) [reply languages/python/oso/oso/oso.py:86]: yes!

**Pull request merged**